task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+  name: fbc-related-image-check
+spec:
+  params:
+    - name: IMAGE_URL
+      description: the fully qualified image name
+  results:
+    - name: HACBS_TEST_OUTPUT
+  workspaces:
+    - name: workspace
+  steps:
+    - name: check-related-images
+      image: quay.io/redhat-appstudio/hacbs-test:latest
+      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      securityContext:
+        runAsUser: 1000
+      env:
+        - name: HOME
+          value: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      resources:
+        limits:
+          memory: 4Gi
+          cpu: 2
+        requests:
+          memory: 512Mi
+          cpu: 10m
+      script: |
+        #!/usr/bin/env bash
+        #Wanted to use extracted content from previous task, but for some ungodly reason you cannot create folder in workdir root directory
+        #unless its an emptydir volume wich is not transitioned to next step, because it is - emptydir
+        #I love tekton <3
+        mkdir confdir
+
+        conffolder=$(cat ../sanity-inspect-image/image_inspect.json | jq -r '.Labels ."operators.operatorframework.io.index.configs.v1"')
+        if ! oc image extract $(params.IMAGE_URL) --path $conffolder/*:confdir/ ; then
+        echo "Unable to extract image! Skipping checking related images!"
+          HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -f 1)"
+          echo "${HACBS_TEST_OUTPUT}" | tee $(results.HACBS_TEST_OUTPUT.path)
+          exit 0
+        fi
+
+        relImgs=$(cat confdir/catalog.yaml | yq '.relatedImages[].image' | sed 's/---//')
+        echo "These are related images: $relimgs"
+        # cycle through those and show outputs
+        for i in ${relImgs// /}
+        do
+          if ! skopeo inspect --no-tags docker://$i; then
+            echo "Skopeo inspect failed on related image: $i"
```

Comment:
So this is error we will show to users? Shouldn't be it rephrased like: `Cannot fetch related image` instead of mentioning skopeo inspect, which is implementation detail?